src/content/blog/2023-review.md

```diff
@@ -54,27 +54,28 @@ categories:
   * Mike Anello interviewed Randy for [DrupalEasy Podcast S15E1 - Randy Fay - DDEV project governance and health](https://www.drupaleasy.com/podcast/2023/06/drupaleasy-podcast-s15e1-randy-fay-ddev-project-governance-and-health).
 
 * Maintainers:
-  * **[Stas Zhuk](./introducing-maintainer-stas.md)** became a fully-privileged maintainer (part-time, paid).
+  * **[Stas Zhuk](/blog/introducing-maintainer-stas/)** became a fully-privileged maintainer (part-time, paid).
   * **Matt Stein** made incredible advances for ddev.com and for the docs while he had availability to do so, but has stepped back (mostly) into the background now. He remains a fully-privileged maintainer.
   * **Simon Gilli** stepped away from the project, and we have lost contact with him.
 
 * DDEV's **testing infrastructure** has been improved with newer Mac ARM64 machines, and is now testing Rancher Desktop and OrbStack as well as Docker Desktop and Colima. (We test every commit on macOS with Docker Desktop, Rancher Desktop, Colima QEMU, Colima VZ, Orbstack, and WSL2 with both Docker Desktop and docker-ce. On traditional WIndows we test with Docker Desktop, on Linux with docker-ce (on both AMD64 and ARM64 architectures)).
 * Randy was awarded the Drupal community's prestigious [Aaron Winborn award](https://www.drupal.org/community/cwg/blog/2023-aaron-winborn-award-winner-randy-fay).
-* Randy [went biking for two months](./randy-in-patagonia.md) in Patagonian Chile and the project continued on as normal! Thanks to all of you!
+* Randy [went biking for two months](/blog/randy-in-patagonia/) in Patagonian Chile and the project continued on as normal! Thanks to all of you!
 
 ## Statistics for the year
 
-* DDEV: [594 PRs merged](https://www.google.com/url?q=https://github.com/ddev/ddev/pulls?q%3Dis%253Apr%2B%2B%2Bmerged%253A2023-01-01..2023-12-31&sa=D&source=editors&ust=1709057329318619&usg=AOvVaw1ichtVuUK-yicP9Y4ZbqVJ). Issues and PRs in many other DDEV repos not shown here.
-* [Amplitude](https://www.google.com/url?q=https://analytics.amplitude.com/ddev/dashboard/jhv7ksg&sa=D&source=editors&ust=1709057329318855&usg=AOvVaw3-YdyXb3I3dwogT7_sTXul) shows about 8000 active weekly DDEV users, up at least 3000 in the last year.
-* 225 [total contributors to DDEV](https://www.google.com/url?q=https://github.com/drud/ddev/graphs/contributors&sa=D&source=editors&ust=1709057329319044&usg=AOvVaw264MR7wU2vWx4uZ0BnaqG0)
-* [422 answers](https://www.google.com/url?q=https://stackoverflow.com/users/215713/rfay?tab%3Danswers%26sort%3Dnewest&sa=D&source=editors&ust=1709057329319223&usg=AOvVaw3XRNxgfJgWIhOzCG-WiPgs) now on Stack Overflow, was 213, reputation increased to 10,750, which means it's easier to edit and support existing questions and answers as things change.
+* DDEV: [594 PRs merged](https://github.com/ddev/ddev/pulls?q=is%3Apr+merged%3A2023-01-01..2023-12-31). Issues and PRs in many other DDEV repos not shown here.
```

Comment:
Thanks, this article started out in google docs and they stick those stupid preambles on there when exporting.